### PR TITLE
Converts alert box link clicks to component library

### PR DESCRIPTION
**Pull request by bsmartin-ep**
## Description

This PR will...

1. Update the _component-library_ version to use new alert box links events
2. Remove manual events from several applications
3. Update Drupal events to use new taxonomy 
4. Add the new event taxonomy to the lookup object

https://github.com/department-of-veterans-affairs/component-library/pull/41
https://github.com/department-of-veterans-affairs/va.gov-team/issues/14489

## Acceptance criteria
- [ ] Are all removed events correctly covered by new component-library events?  Are all headlines _etc_ correct?

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by jonwehausen — APPROVED** [review]

File: src/applications/gi/components/profile/SchoolClosingDetails.jsx [hunk 1]
```diff
@@ -23,17 +22,7 @@ const SchoolClosingDetails = ({
...
-                href={schoolWebsite}
-                target="_blank"
-                rel="noopener noreferrer"
-                onClick={() => {
-                  recordEvent({
-                    event: 'nav-warning-alert-box-content-link-click',
-                    alertBoxHeading: headline,
-                  });
-                }}
-              >
```
Comment (zurbergram) [inline src/applications/gi/components/profile/SchoolClosingDetails.jsx:36]: where is `event` or `alertBoxHeading` being set in https://github.com/department-of-veterans-affairs/component-library/blob/master/src/helpers/analytics.js or https://github.com/department-of-veterans-affairs/component-library/blob/master/src/components/AlertBox/AlertBox.jsx ?

Comment (bsmartin-ep) [reply src/applications/gi/components/profile/SchoolClosingDetails.jsx:36]: @zurbergram - the component triggers a CustomEvent on the document.body....

https://github.com/department-of-veterans-affairs/component-library/blob/eaffb602607f3d1b021d0eb2f9101e7c05dc3b28/src/components/AlertBox/AlertBox.jsx#L62

...which is then interpreted into an actual dataLayer event (with the headline) here...

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/component-library-analytics-setup.js

**Review by zurbergram — APPROVED** [review]

**Review by ncksllvn — APPROVED** [review]

**Review by timwright12 — APPROVED** [review]

**Review by Mottie — APPROVED** [review]

**Review by jonwehausen — APPROVED** [review]
LGTM

**Review by jbalboni — APPROVED** [review]

**Review by jonwehausen — COMMENTED** [review]
LGTM

**Pull request merged**